<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtUserSearchFieldsPage</class>
 <widget class="QWizardPage" name="QtUserSearchFieldsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="instructionsLabel_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nickInputLabel_">
     <property name="text">
      <string>Nickname:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nickInput_"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="firstInputLabel_">
     <property name="text">
      <string>First name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="firstInput_"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lastInputLabel_">
     <property name="text">
      <string>Last name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lastInput_"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="emailInputLabel_">
     <property name="text">
      <string>E-Mail:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="emailInput_"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="fetchingThrobber_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="fetchingLabel_">
     <property name="text">
      <string>Fetching search fields</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QWidget" name="formContainer_" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
